src/components/Pages/Friends/friends.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import './friends.scss';
-// import { homedir } from 'os';
 
```

Comment:
Make sure to completely remove commented out or unused code

Reply:
Commented code is gone